### ISSUE TITLE
Fine-grained correlation loss calculation

### ISSUE THREAD

**Issue by liumc14** [RUCAIBox/REAR#12]
In the paper, the fine-grained correlation loss calculation function seems to be a pairwise calculation between the predicted correlations of k documents, but in the actual code, why is the calculation between predictions and labels? Can you help me answer this question? Thanks
![微信图片_20240519173206](https://github.com/RUCAIBox/REAR/assets/93915930/0ebaa217-da0a-49bb-ad3c-f243e7bd1e2b)

https://github.com/RUCAIBox/REAR/blob/73eb821581c27b7eed2ed05bf998b98f20041091/rear/src/modeling_rearllama.py#L62

**Comment by yhao-wang** [comment]
Hello, thank you for following our work. We optimize the difference in logits based on the difference in labels (int (label_1>label_2))
